src/main/java/com/cognifide/aemrules/checks/HtlAttributesShouldBeAtTheEndCheck.java

```diff
@@ -0,0 +1,67 @@
+/*-
+ * #%L
+ * AEM Rules for SonarQube
+ * %%
+ * Copyright (C) 2015 Cognifide Limited
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.cognifide.aemrules.checks;
+
+import com.cognifide.aemrules.htl.checks.AbstractHtlCheck;
+import com.cognifide.aemrules.tag.Tags;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.sling.scripting.sightly.compiler.expression.Expression;
+import org.apache.sling.scripting.sightly.impl.compiler.Syntax;
+import org.sonar.check.Priority;
+import org.sonar.check.Rule;
+import org.sonar.plugins.html.node.Attribute;
+import org.sonar.plugins.html.node.TagNode;
+
+@Rule(
+    key = HtlAttributesShouldBeAtTheEndCheck.RULE_KEY,
+    name = HtlAttributesShouldBeAtTheEndCheck.RULE_MESSAGE,
+    priority = Priority.MINOR,
+    tags = Tags.AEM
+)
```

Comment:
Please merge `master` branch to your branch and implement Technical Debt using `Metadata` annotation.

Reply:
done